examples/serialization/compact_serialization_example.py

```diff
@@ -1,5 +1,10 @@
 from hazelcast import HazelcastClient
-from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader
+from hazelcast.serialization.api import (
+    CompactSerializer,
+    CompactWriter,
+    CompactReader,
+    CompactSerializableClass,
```

Comment:
Don't forget to change this file if you change `get_class` method's return type.

Reply:
Removed the import for that at: https://github.com/hazelcast/hazelcast-python-client/pull/546/commits/6f6d8fdd8681b6349a84526961a55ef96e1a4eed